goby.go

```diff
@@ -63,7 +64,15 @@ func main() {
 			return
 		}
 
-		v, err := vm.New(dir, args)
+		var v *vm.VM
+
+		if *issueOptionPtr {
+			fmt.Println("Will report first issue...\n")
```

Comment:
We should say `Will generate full issue information` or something like that. We don't really help users report issues.